src/main/java/org/sefglobal/scholarx/util/ProgramStatus.java

```diff
@@ -10,9 +10,9 @@
     COMPLETED,
     REMOVED;
 
-    private static ProgramState[] states = values();
+    private static ProgramStatus[] statuses = values();
```

Comment:
Hi, it's better to name the variable statusList/statusArray. Since status and statuses are both plural nouns. We will focus on using status for both singular and plural occurrences.

Reply:
done

Reply:
Hi this seems like not fixed yet, can you please fix this?